tools/jbrowse/jbrowse_to_container.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<tool id="jbrowse_to_container" name="Add organisms to JBrowse container" version="@WRAPPER_VERSION@">
+    <description>will replace any existing organism</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+
+        if [ -z "@DATA_DIR@" ]; then
+            echo '@DATA_DIR@ variable must be set to use this tool';
+            exit 1;
+        fi
+
+        &&
+
+        rm -rf "@DATA_DIR@/"*
+
+        &&
+
+        #for $org in $organisms:
+            #if $org.advanced.unique_id:
+                #set dataset_name = $org.advanced.unique_id
+            #else:
+                #set dataset_name = $org.jbrowse.id
+            #end if
+
+            cp -R '${org.jbrowse.extra_files_path}/data/' "@DATA_DIR@/${dataset_name}"
+
+            &&
+
+            echo -e "[general]\ndataset_id  = ${dataset_name}" >> "@DATA_DIR@/${dataset_name}/tracks.conf"
+
+            &&
+        #end for
+
+        cp $trackList "@DATA_DIR@/datasets.conf"
+
+        &&
+
+        python $__tool_directory__/jbrowse_to_container.py @JBROWSE_BASE_URL@ > $output
+      ]]></command>
+    <configfiles>
+        <configfile name="trackList">
+            <![CDATA[
+#for $org in $organisms:
+#if $org.advanced.unique_id:
```

Comment:
Yep, I would have liked, but jbrowse don't like it at all, for some strange reason it fails parsing when there are spaces/tabs...